ipa_room_exploration/ros/src/room_exploration_action_client.cpp

```diff
@@ -36,14 +34,37 @@ std::ostream& operator<<(std::ostream& os, const geometry_msgs::Pose2D& obj)
 int main(int argc, char **argv)
 {
 	ros::init(argc, argv, "room_exploration_client");
-	ros::NodeHandle nh;
+	ros::NodeHandle priv_nh("~");
 
 	actionlib::SimpleActionClient<ipa_building_msgs::RoomExplorationAction> ac("room_exploration_server", true);
 
-	// read in test map
-	const std::string test_map_path = ros::package::getPath("ipa_room_segmentation") + "/common/files/test_maps/";
```

Comment:
Please keep these lines in a comment, since some users do not run a map server or use our internal map storage format. In that case they load their map from our examples like this.